src/Routes/RosSystemDetail/RosSystemDetail.js

```diff
@@ -105,6 +118,7 @@ class RosSystemDetail extends React.Component {
                                         hideBack
                                         showDelete={ false }
                                         hideInvDrawer
+                                        LastSeenWrapper={() => <div>{report_date}</div>} /* eslint-disable-line camelcase */
```

Comment:
```suggestion
                                        
                                            hideBack
                                            showDelete={ false }
                                            hideInvDrawer
                                            className='rosDetailsHead'
                               
```


To hide `last seen` entry apply class `rosDetailsHead` and add below CSS changes to this class in file - https://github.com/RedHatInsights/ros-frontend/blob/main/src/Routes/RosSystemDetail/ros-details-page.scss

```

.rosDetailsHead {
  .ins-entity-detail {
    .ins-entity-facts {
      .pf-l-flex:last-child {
        display: none !important;
      }
    }
  }
}

```

And remove this `LastSeenWrapper`.

Reply:
@karelhala  - currently I don't see an option to hide default list of facts info on detail page. 
In case of ROS service, `last reported` date is more relevant than `last seen` so we would like to hide `last seen` fact wrapper from detail header. 
Will it be OK to hide using above class name and CSS change or is there any better way to hide any fact entry?

As per my understanding, as I am suggesting to use `last-child`, it will break on adding new entry to facts details on `InventoryDetailHead` component side.

Reply:
Thanks @kgaikwad! Yeah, It seems fine to me 👍🏻